src/app/components/Button/Button.module.css

```
@@ -0,0 +1,28 @@
+.submit {
+  background-color: var(--main-font-color);
+  color: var(--secondary-font-color);
+  font: var(--main-font);
+  border: none;
+  border-radius: 0.25em;
+  padding: 1em 5em;
+}
+
+.submit:active {
+  background-color: var(--addButton-active);
```

If you use a more generic name, you could reuse it in other components.
```suggestion
  background-color: var(--main-active-color);
```